lib/index.ts

```diff
@@ -46,6 +47,19 @@ const nodeSdk = ({ managementKey, publicKey, ...config }: NodeSdkArgs) => {
       'x-descope-sdk-node-version': process?.versions?.node || '',
       'x-descope-sdk-version': BUILD_VERSION,
     },
+    hooks: {
+      ...config.hooks,
+      beforeRequest: [
+        (requestConfig) => {
+          if (!requestConfig.token && authManagementKey) {
+            // eslint-disable-next-line no-param-reassign
+            requestConfig.token = authManagementKey;
```

Comment:
The auth management key needs to be appended to the end of the bearer both when we have a refresh JWT and when we don't. Unless I'm missing something - I don't think this code will achieve this